src/main/java/com/mongodb/hibernate/OnlyForTest.java

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
Is it necessary to commit these two files?

Reply:
not necessary. I think we could tell whether CI testing works by log without a concrete testing class.